model_analyzer/reports/report_manager.py

```diff
@@ -472,42 +472,38 @@ def _create_summary_config_phrase(self, best_run_config,
         else:
             return f"{num_configurations} configurations, {config_names_str}"
 
-    def _create_summary_throughput_phrase(self, report_key,
-                                          best_run_config_measurement,
-                                          multi_model):
-        default_throughput = round(
-            self._find_default_configs_throughput(report_key))
-        best_throughput = round(
-            best_run_config_measurement.get_non_gpu_metric_value(
-                'perf_throughput'))
-        throughput_gain = round(
-            (best_throughput - default_throughput) / default_throughput *
-            100) if default_throughput else None
-
-        throughput_phrase = "total " if multi_model else ""
-        throughput_phrase = (
-            throughput_phrase +
-            f"throughput: <strong>{best_throughput} infer/sec</strong>.<br><br>"
-        )
+    def _create_summary_objective_phrase(self, report_key,
```

Comment:
can we type check this?

Reply:
We can! (and it should have been). Type checking added.